### PR TITLE
Send message to security channel on segfault label

### PR DESCRIPTION
**Pull request by mkindahl**
If an issue is labeled with either `segfault` or `security`, send a
message to the security channel that the issue needs attention.

Disable-check: force-changelog-file

### CODE REVIEW

**Review by thanasisk — APPROVED** [review]

**Review by jnidzwetzki — APPROVED** [review]
Added a few minor comments. But nothing that should block merging.

File: .github/workflows/issue-handling.yaml [hunk 1]
```diff
@@ -29,6 +29,26 @@ jobs:
...
+  notify-sec:
+    name: Notify security channel
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issues' && contains(github.event.issues.issue.labels.*.name, 'segfault')
+    env:
+      SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+      SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL_SECURITY }}
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
```
[the remark this answers is not in the document]
Comment (mkindahl) [reply .github/workflows/issue-handling.yaml:41]: We don't

Comment (mkindahl) [inline .github/workflows/issue-handling.yaml:41]:
```suggestion
```

File: .github/workflows/issue-handling.yaml [hunk 2]
```diff
@@ -29,6 +29,25 @@ jobs:
           github-token: ${{ secrets.ORG_AUTOMATION_TOKEN }}
           labeled: continuous_aggregate
 
+  notify-sec:
+    name: Notify security channel
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issues' && contains(github.event.issues.issue.labels.*.name, 'segfault')
```
Comment (svenklemm) [inline .github/workflows/issue-handling.yaml:35]: should probably also trigger on security

Comment (mkindahl) [reply .github/workflows/issue-handling.yaml]: Updated.

**Pull request merged**